src/components/DetailsMetrics/detailsMetrics.utils.js

```diff
@@ -89,3 +89,109 @@ export const generateMetricsItems = metrics => {
     })
     .value()
 }
+
+export const getTitle = fullName =>
+  fullName.substring(fullName.lastIndexOf('.') + 1).replace(/-/g, ' ')
+
+function formatTime(dates) {
+  const options = {
+    hour: 'numeric',
+    minute: '2-digit',
+    hour12: true
+  }
+  return dates.map(date => new Date(date).toLocaleTimeString('en-US', options))
+}
+
+function formatDate(dates) {
+  const options = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit'
+  }
+  return dates.map(date => new Date(date).toLocaleDateString('en-US', options).replace(/\//g, '.'))
+}
+
+// TODO: add support for Any time selection
+function isRangeOverOneDay(values, dateRange) {
+  const dates = values.map(([date]) => date)
+  const startDate = new Date(dateRange.start)
+  const endDate = new Date(dateRange.end)
+  const differenceInDays = Math.abs((endDate - startDate) / (1000 * 60 * 60 * 24))
+  return differenceInDays > 1 ? formatDate(dates) : formatTime(dates)
+}
+
+export const formatNumber = num => {
+  if (num >= 1e6) {
+    return (num / 1e6).toFixed(0) + 'M'
+  } else if (num >= 1e3) {
+    return (num / 1e3).toFixed(0) + 'k'
+  } else {
+    return num.toString()
+  }
+}
+
+// TODO: verify with Jonathan the drift status option
+const driftStatusConfig = {
```

Comment:
Use color vars as we do today for the drift indicator

Reply:
done.